### DISCUSSION TITLE
Dropdown tree multiselect

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#55] [Ideas]
We have the ability now as a plugin to have a checkbox in the dropdown select, and was wondering how difficult it would be to structure the dataset of the dropdown to enable a tree structure with checkboxes for multiple selection.

eg.
https://ej2.syncfusion.com/javascript/demos/#/material/drop-down-tree/checkbox.html

**Comment by oyejorge** [comment]
Hmm, first thought is this might be a little difficult. But option groups do give us the hierarchy already, they just don't have form values...
